src/components/RangeSelector/RangeSelector.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { Button, Container, ButtonContainer } from "./RangeSelector.styles";
+
+const range = {
+  day1: {
+    range: 1,
+    text: "1d",
+    id: Math.random(),
```

Comment:
you can rename `text` to value and use it as an `id` because it's not going to appear twice